packages/smooth_app/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```

Comment:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-all.zip
```

Reply:
What would change

Reply:
nothing I guess, it's good to keep Gradle up to date in general ?